README.md

```diff
@@ -109,6 +98,5 @@ $ curl http://example.com/text-generator -X POST -H "Content-Type: application/j
 
 ## Get started
 
-* [Read the docs](https://docs.cortex.dev)
-* [Report an issue](https://github.com/cortexlabs/cortex/issues)
-* [Join our community](https://gitter.im/cortexlabs/cortex)
+* [Cortex Core](https://docs.cortex.dev/core) - free and open source community offering.
+* [Cortex Cloud](https://docs.cortex.dev/cloud) - managed Cortex clusters on AWS and GCP.
```

Comment:
I think Cortex Cloud might deserve an H2 section in the main readme. We can discuss live